### PR TITLE
Add github bulk import

### PR DESCRIPTION
**Pull request by claireriley**
Added section for release coming up early next week.

@umeshp7 I used the screenshot with the failed import as a placeholder—do you mind sending me a fully successful screenshot?

Also blurred out "beta" mentions 😅 can update that once in prod

### CODE REVIEW

**Comment by umeshp7** [comment]
Yes, I will send the screenshot for successful Import completed screen.

File: src/pages/docs/getting-started/importing-and-exporting-data.md [hunk 1]
```diff
@@ -106,6 +108,32 @@ When importing into a team workspace, you can also choose to add the APIs to the
...
+
+In Postman, select your GitHub organization, repository, and branch, then **Continue**.
+
+<img alt="Select org, repo, branch" src="https://assets.postman.com/postman-docs/select-repo.jpg"/>
+
+Confirm the files you would like to import into Postman. You can also opt to **Generate collection from imported APIs** and select what you would like to link this collection as. Click **Show advanced settings** to control how Postman should generate collections based on your file types, then select **Import**.
+
+<img alt="Confirm github import" src="https://assets.postman.com/postman-docs/confirm-import.jpg"/>
+
+You will receive confirmation once the import has completed.
```
Comment (sowmyaraj92) [inline src/pages/docs/getting-started/importing-and-exporting-data.md:129]: I'd change "receive confirmation" to "receive a confirmation once the import is complete"

**Review by sowmyaraj92 — CHANGES_REQUESTED** [review]
Hey @claireriley! This looks great 🥳 Just one tiny suggestion.

File: src/pages/docs/getting-started/importing-and-exporting-data.md [hunk 2]
```diff
@@ -46,6 +46,8 @@ Postman can import and export Postman data, including collections, environments,
 
     * [Importing API specifications](#importing-api-specifications)
 
+    * [Importing GitHub repositories](#importing-github-repositories)
```
Comment (umeshp7) [inline src/pages/docs/getting-started/importing-and-exporting-data.md:49]: Should this be "Importing `via` GitHub repositories" ? @claireriley

**Comment by umeshp7** [comment]
Added a small comment, see if it makes to you. Everything else looks good to me! 🎉 @claireriley

**Review by sowmyaraj92 — APPROVED** [review]
Lgtm! 🥳👏🏻

**Pull request merged**